### PR TITLE
Update readme, add contributing guide & changelog

### PR DESCRIPTION
**Pull request by draffensperger**
The usage instructions will continue to evolve as the library matures, but this is a decent start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,151 @@
...
+[![Known Vulnerabilities][snyk-image]][snyk-url]
+[![Apache License][license-image]][license-url]
 
-OpenCensus Web is an experimental implementation of OpenCensus, a toolkit for
-collecting application performance and behavior monitoring data.
+OpenCensus Web is an implementation of OpenCensus, a toolkit for collecting
+application performance and behavior monitoring data. This library currently
+generates traces for the initial page load of a site in the browser. It supports
+sending the trace sampling decision and trace ID from the web server to the
+browser client to enable latency debuging across the stack.
```
Comment (mayurkale22) [inline README.md]: Nit: `s/debuging/debugging`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,151 @@
...
+<script>ocAgent = 'https://example.com/my-opencensus-agent-endpoint'</script>
+```
+
+### 3. Optional: Send a trace parent and sampling decision from your server
+
+Currently, the OpenCensus Web will sample all requests by default. This is
+useful for experimentation with the library but is not appropriate for a real
+application.
+
+OpenCensus Web also supports connecting the server side spand for the initial
```
Comment (mayurkale22) [inline README.md]: typo: spand?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,151 @@
...
+first mark the existing API as deprecated and keep it for 18 months before removing it.
+
+**Beta**: Libraries defined at a Beta quality level are expected to be mostly stable and we're
+working towards their release candidate. We will address issues and requests with a higher priority.
+There may be backwards incompatible changes in a minor version release, though not in a patch
+release. If an element is part of an API that is only meant to be used by exporters or other
+opencensus libraries, then there is no deprecation period. Otherwise, we will deprecate it for 18
+months before removing it, if possible.
+
+**Alpha**: Libraries defined at a Alpha quality level can be unstable and could cause crashes or data loss. Alpha software may not contain all of the features that are planned for the final version. The API is subject to change.
```
Comment (mayurkale22) [inline README.md]: Nit: `s/a Alpha/an Alpha`?

Comment (draffensperger) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (draffensperger) [reply README.md]: Fixed, "spans"

File: README.md (hunk 1 above)
Comment (draffensperger) [reply README.md]: Thanks, done

**Pull request merged**